templates/index.html

```diff
@@ -9,26 +9,54 @@ <h1 class="display-4">To-Do App</h1>
 
 <div class="row">
   <div class="col-md-12">
-    <h2>To-Do list...</h2>
+    <h2>Add Item</h2>
+    <form action="{{ url_for('index') }}" method="post" class="mb-3">
+      <div class="form-group">
+        <label for="title">Item Title:</label>
+        <input type="text" id="title" name="item_title" required class="form-control" />
+      </div>
+      <div class="form-group">
+        <label for="description">Item Description:</label>
+        <input type="text" id="description" name="item_description" class="form-control" />
+      </div>
+      <div class="input-group-append">
+        <button class="btn btn-primary" type="submit">Add Item</button>
+      </div>
+    </form>
+
+    <br>
+    <h2>To-Do List:</h2>
+
     <ul class="list-group mb-4">
-      <!-- List the todo items here -->
-      <table style="width:100%">
-        <tr>
-          <!--<th>ID</th>-->
-          <th>Title</th>
-          <th>Status</th>
-        </tr>
-        {% for item in items %}
-        <tr>
-          <!--<td><a href="{{url_for('read_item', id=item.id)}}">{{ item.id }}</a></td>-->
-          <td><a href="{{url_for('read_item', id=item.id)}}">{{ item.title }}</a></td>
-          <td>{{ item.status }}</td>
-        </tr>
-        {% endfor %}
-      </table>
-      <hr>
+      {% for item in items %}
+      <li class="list-group-item">
+        <div class="row">
+          <span class="col"><b>{{ item.title }}</b>: {{ item.status }}</span>
+          <div class="col-auto">
+            <form action="{{ url_for('delete_item', id=item.id) }}" method="post" class="d-inline">
+              <button type="submit" class="btn btn-danger">Delete</button>
+            </form>
+            {% if item.status == 'Not Started' %}
+            <form action="{{ url_for('in_progress_item', id=item.id) }}" method="post" class="d-inline">
```

Comment:
I'd indent these one level in to show they're contained within the `if` block

Reply:
Good point! I've indented these by one level